.changeset/violet-planets-impress.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/vertexai': minor
```

Comment:
`'firebase': minor`, as the changeset checker is yelling about.

Reply:
done